core/domain/takeout_service_test.py

```diff
@@ -446,9 +447,9 @@ def set_up_trivial(self):
 
     def test_export_nonexistent_user(self):
         """Setup for nonexistent user test of export_data functionality."""
-        with self.assertRaises(
-            user_models.UserSettingsModel.EntityNotFoundError):
-            takeout_service.export_data_for_user('fake_user_id')
+        with self.assertRaises(user_models.UserSettingsModel
+                               .EntityNotFoundError):
+            user_models.UserSettingsModel.export_data('fake_user')
```

Comment:
Why is this line changed from what it was before? Please keep it as it was, since this file is supposed to test takeout_service and not user_models.

Reply:
Done